app/controllers/concerns/report_embeddable.rb

```diff
@@ -2,6 +2,7 @@
 module ReportEmbeddable
   # sets up the @report_data structure which will be converted to json
   def build_report_data(options = {})
+    @report = Rails.cache.read(@report.cache_key_with_responses(current_mission))
```

Comment:
I think cache_key_with_responses probably belongs in ReportEmbeddable. It seems like Report shouldn't need to know about Response and I18n.